doc/source/ray-core/memory-management.rst

```diff
@@ -304,7 +304,35 @@ To enable object spilling to remote storage (any URI supported by `smart_open <h
             "max_io_workers": 4,  # More IO workers for remote storage.
             "min_spilling_size": 100 * 1024 * 1024,  # Spill at least 100MB at a time.
             "object_spilling_config": json.dumps(
-                {"type": "smart_open", "params": {"uri": "s3:///bucket/path"}},
+                {
+                  "type": "smart_open", 
+                  "params": {
+                    "uri": "s3:///bucket/path"
+                  },
+                  "buffer_size": 100 * 1024 * 1024 # Use a 100MB buffer for writes
+                },
+            )
+        },
+    )
+
+It is recommended that you specify a large buffer size (> 1MB) to lower IO requests during spilling.
+
+Spilling to multiple remote storages is also supported.
+
+.. code-block:: python
+
+    ray.init(
+        _system_config={
+            "max_io_workers": 4,  # More IO workers for remote storage.
+            "min_spilling_size": 100 * 1024 * 1024,  # Spill at least 100MB at a time.
+            "object_spilling_config": json.dumps(
+                {
+                  "type": "smart_open", 
+                  "params": {
+                    "uri": ["s3:///bucket/path1", "s3:///bucket/path2, "s3:///bucket/path3"]
```

Comment:
Are you sure it's three slashes? The smart_open example shows two slashes (https://github.com/RaRe-Technologies/smart_open) like `'s3://commoncrawl/`

Reply:
Oops, was following the previous example. Fixed this and the original one.